app/grants/templates/grants/index.html

```diff
@@ -47,9 +47,7 @@ <h1 class="grants-hero__title mb-2">{% trans 'Dev Grants' %}</h1>
           support for some of the best OSS projects in the space.
           {% endblocktrans %}
         </p>
-        <a class="button button--primary font-weight-bold" href="{% url 'grants:new' %}">
-          {% trans 'Create a Grant' %}
```

Comment:
Do we no longer want the `Create a Grant` button here?

Reply:
We do. Not sure why it got deleted... Will add back in.